MyCapytain/resources/collections/cts.py

```diff
@@ -185,30 +185,30 @@ def parse(resource, parent=None):
         :param resource: Element rerpresenting a work
         :param type: basestring, etree._Element
         :param parent: Parent of the object
-        :type parent: TextGroup
+        :type parent: XmlCtsTextgroupMetadata
```

Comment:
Should Group be capitalized here? I.e., XmlCtsTextGroupMetadata? Maybe not worth the trouble. Or maybe not even desired.